readme.md

```diff
@@ -179,6 +179,12 @@ a script, [`manage.py`](/manage.py). The most important command is:
 
 [Alembic]: https://alembic.readthedocs.org/
 
+#### Browsers blacklisting
+
+We provide you with basic browsers blacklisting so you can exclude specific browser from accessing your application.
```

Comment:
I don't like the wording / style here. Avoid using "we", you're not making a speech. Suggested:

> ```
There's a basic browser detection / blacklisting feature, which handles a common requirement on many projects.
It can be used to serve an "unsupported browser" page for browsers / devices that are explicitly blacklisted in the Technical Specification and/or Statement of Work.

Use the `backend.utils.ua.is_supported` function to match the user agent (this is already done for the landing page handler).
You can edit [`browsers.yml`](/browsers.yml) to tweak the blacklist.
```